content/en/blog/2023/notary-enlightning-podcast.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Notary Project featured on the Enlightning Podcast
+author:  "Notary Project maintainers"
+date: 2023-10-04
+draft: false
+---
+
+Notary Project maintainers Toddy Mladenov and Milind Gokarn talk with host Whitney Lee about the Notary Project on the [Enlightning Podcast](https://tanzu.vmware.com/content/videos/enlightning-ensuring-software-authenticity-introduction-to-notary-project). This wide-ranging discussion covers everything from the project history, to concepts, and to the future of the project.
```

Comment:
@zr-msft 
How about embedding the Youtube video player into this page? It brings a convenient experience to watch the video at notaryproject.dev. You can try embed the video by using the following code:

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/FwpHZfNE9LQ?si=pzxe-qRM2z3oetwV" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Reply:
good call! fixed